packages/slice-machine/components/Flex/index.tsx

```diff
@@ -22,10 +20,11 @@ export const Flex = ({
   </Box>
 );
 
-export const Col = ({
-  children,
-  cols = 2,
-}: {
+interface ColProps {
   children: React.ReactNode;
   cols?: number;
-}) => <Box sx={{ flex: `0 ${100 / cols - 1}%`, mb: 1 }}>{children}</Box>;
+}
+
+export const Col: React.FC<ColProps> = ({ children, cols = 2 }) => (
```

Comment:
You know you can pass the propTypes like 
```
FC<{{ children: React.ReactNode; cols?: number;}>
```

Reply:
Yes :) it's just that i don't like this way.
Because If i need the type of those properties elsewhere then it's not re-usable.
That's a personal preference that I consider best practice for code re-usability 😄 .

Reply:
True